src/pages/apj-nerd-days.js

```diff
@@ -214,7 +224,7 @@ const NerdDaysPage = () => {
                     }
                   `}
                 >
-                  <CtaItem date="Nov 11, 2020" to="/apj-nerd-days">
+                  <CtaItem date="Nov 11, 2020" to="/emea-nerd-days">
```

Comment:
It looks like the `AMER` registration link is missing from this page. Do you mind adding that as well?